### PR TITLE
Small improvement for Moves::ImportPeople. More need to be done!

### PR DESCRIPTION
**Pull request by alexdesi**
### What?

`Moves::ImportPeople` is spaghetti 🍝  code,
I need to work on that part at the moment,  so this is just a mini step to clean it up with minimal risk.

### CODE REVIEW

File: app/services/moves/import_people.rb [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 module Moves
   class ImportPeople
-    attr_accessor :items
+    attr_accessor :people_nomis_prison_numbers
```
Comment (martyn-w) [inline app/services/moves/import_people.rb:5]: this name doesn't roll of the tongue easily, how about `nomis_prisoner_numbers` ?

File: app/controllers/api/v1/people_controller.rb [hunk 2]
```diff
@@ -74,7 +74,7 @@ def json_api_errors_for(error)
       def import_person_from_nomis prison_number
```
Comment (martyn-w) [inline app/controllers/api/v1/people_controller.rb:74]: tiny point: could we use parentheses in method definitions with parameters, i.e.  `def import_person_from_nomis(prison_number)`

Comment (alexdesi) [reply app/controllers/api/v1/people_controller.rb:74]: yes, I agree

File: app/services/moves/import_people.rb (hunk 1 above)
Comment (alexdesi) [reply app/services/moves/import_people.rb:5]: :D yeah prison_numbers is enough

**Pull request merged**